.changeset/public-snakes-fetch.md

```diff
@@ -0,0 +1,11 @@
+---
+"@fluidframework/tree": minor
+"fluid-framework": minor
+"__section": tree
+---
+TreeNodes now implicitly generate identifiers on access instead of throwing
+
+Accessing a defaulted [identifier](https://fluidframework.com/docs/api/fluid-framework/schemafactory-class#identifier-property) on an [Unhydrated](https://fluidframework.com/docs/api/fluid-framework/unhydrated-typealias) `TreeNode` no longer throws a usage error.
```

Comment:
Nit: a quick code example would probably be helpful (I'm pushing for more of these generally, as I think they make a huge difference)

Reply:
Done.